### ISSUE TITLE
ImageNet Evaluation

### ISSUE THREAD

**Issue by SheldonHS** [Alpha-VL/ConvMAE#9]
Thanks for sharing the great work. 
I encountered difficulties in reproducing the evaluation results on [FINETUNE.md](https://github.com/Alpha-VL/ConvMAE/blob/main/FINETUNE.md). My evaluation results are:
`* Acc@1 1.090 Acc@5 2.188 loss 8.955`
`Accuracy of the network on the 50000 test images: 1.1%`
That's obviously too big a gap.

I download the ImageNet-1K following your guidance and prepared the ImageNet-1K following [Jasonlee1995](https://github.com/Jasonlee1995/ImageNet-1K). Are there any details I haven't noticed, or any specific requirements for preparing the dataset？

**Comment by TeleeMa** [comment]
@SheldonHS Could you provide more details about your experiment?
1. Did you do evaluation with the fine-tuned weights of ours or weights you fine-tuned by yourself.
2. If you utilized provided fine-tuned weights, make sure that you had downloaded the right model in https://drive.google.com/file/d/19F6vQUlITpzNLvXLKi5NRxRLOmKRxqFi/view.
3. If you fine-tuned the model by yourself, please provide more training details like the configs.

**Comment by SheldonHS** [comment]
@TeleeMa Thanks for your reply.
I utilized the provided fine-tuned weights. For the sake of insurance, I downloaded and tested it again, but the result is still the same.
I experimented strictly following [FINETUNE.md](https://github.com/Alpha-VL/ConvMAE/blob/main/FINETUNE.md), except for the CUDA version. The test program runs on three Tesla V100.

**Comment by TeleeMa** [comment]
@SheldonHS Then could you provide the evaluation log for me to check?

**Comment by SheldonHS** [comment]
@TeleeMa Thank you very much for your patience. I've checked many times before opening this issue, so please forgive my bother. Here's the log. 
[ImageNetEvaluation.log](https://github.com/Alpha-VL/ConvMAE/files/8957227/ImageNetEvaluation.log)

**Comment by TeleeMa** [comment]
@SheldonHS I downloaded the codes and checkpoint just now and tested the evaluation, here is my [log.txt](https://github.com/Alpha-VL/ConvMAE/files/8957935/log.txt). Also, I tested with different number of gpus and batch sizes, that made no differences to the final results. Therefore, would you mind comparing your log with mine firstly, and any further questions are welcomed. PS. Don't forget check the ImageNet dataset as we use the same format with DeiT and MAE.

**Comment by SheldonHS** [comment]
@TeleeMa Thanks a lot. It turns out to be the dataset. I tried another way to prepare IN-1k and the results are finally back on track.
